bumpversion/cli.py

```diff
@@ -370,17 +396,32 @@ def main(original_args=None):
             replace=known_args.replace,
             part_configs=part_configs,
         )
-    except sre_constants.error as e:
+    except sre_constants.error:
         # TODO: use re.error here mayhaps, also: should we log?
         sys.exit(1)
+    return vc
+
+
+def _parse_current_version(current_version, vc):
```

Comment:
How about handling `None` in `VersionConfig.parse` and getting rid of this method?